core/config/locales/en.yml

```diff
@@ -1282,7 +1282,7 @@ en:
     inventory: Inventory
     inventory_adjustment: Inventory Adjustment
     inventory_canceled: Inventory canceled
-    inventory_error_flash_for_insufficient_quantity: An item in your cart has become unavailable.
+    inventory_error_flash_for_insufficient_quantity: "'#{names}' has become unavailable."
```

Comment:
I was thinking about that today as well. Initially, I had it wrapping each product name but @tvdeyen suggested moving the apostrophe to the translation file. I've also been slightly bothered by the `has` vs `have` not being changed properly. Thoughts?